src/eduid_webapp/reset_password/settings/common.py

```diff
@@ -63,3 +63,6 @@ class ResetPasswordConfig(FlaskConfig):
     chpass_timeout: int = 600
     # VCCS URL
     vccs_url: str = ''
+    # URL to get the js app that can drive the process to reset the password
+    password_reset_link: str = 'http://login.eduid.se/reset-password/'
```

Comment:
http:// -> https://

Reply:
I am being very picky but I know we will sooner or later have this string in prod by mistake :)